mollie_subscription_ept/controllers/portal.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from odoo.http import request
+from odoo.addons.portal.controllers.portal import CustomerPortal, pager as portal_pager
+from odoo import http, _
+from odoo.exceptions import AccessError, MissingError
+
+
+class CustomerPortalExt(CustomerPortal):
+
+    def _prepare_portal_layout_values(self):
+        values = super(CustomerPortalExt, self)._prepare_portal_layout_values()
+
+        partner = request.env.user.partner_id
+        SaleOrder = request.env['sale.order']
+        subscription_count = SaleOrder.search_count(
```

Comment:
No need to have a separate assignment of sale.order object , it can be directly done as 
subscription_count =request.env['sale.order'].search_count